core/domain/exp_jobs_one_off.py

```diff
@@ -51,13 +54,39 @@ def entity_classes_to_map_over(cls):
     @staticmethod
     def map(exploration_model):
         if not exploration_model.deleted:
-            exp_services.create_exploration_summary(exploration_model.id)
+            exp_services.create_exploration_summary(
+                exploration_model.id, contributor_id_to_add=None)
```

Comment:
It is a bit confusing, but I think that's due to the naming and the choice of args (which I think might be why I instinctively preferred a list before). I mean, when we do update_exploration(), we pass in a contributor id and that's never a problem; the issue is that, in this case, we have this odd "contributor_id_to_add" construct which may be None or may be a committer_id and that's a bit unusual.

I think it's OK, though. The alternative of using a default arg is going to just hide possible errors and I think it's more important for something to fail visibly when one arg is passed in and not the other.